### ISSUE TITLE
E2E Test - Add a user to a Group

### ISSUE THREAD

**Issue by calvinckho** [calvinckho/restvo-app#113]
@davidloesch1 If you can write a e2e test that mimic a user inviting another user to a group

npm install to upgrade to protractor 7.0.0. 
npm run e2e to see it the new dynamic waiting at work.  

1. begin at /app/activity/5e0012f714001a7dbf712de2 (showfeature)
2. click on Sign In
3. log in
4. Click on 'Add User' on top right corner (should open up editparticipants.page.html modal view)
5. Click on 'Add' again, select Mentee
6. search for Asia Ho
7. choose Asia Ho
8. click Done
9. on the Invite to E2E Sample Mentoring screen, there should be a count of 2 users
10. Select Asia Ho again
11. Click Remove button
12. Select Mentor, Click OK
13. On the Confirm Modal, click Remove
14. on the Invite to E2E Sample Mentoring screen, there should be a count of 1 user

**Comment by calvinckho** [comment]
@calixo888 I have done some work to improve upon yours. please check out calvin-add-user. I am able to get it to work but there is a timeout error I cannot isolate yet.

**Comment by calvinckho** [comment]
It is finally working! @calixo888 I identified in line 67:

await showfeature.clickElement('#add-user-to-group');

It is clicking on both the authenticated and unauthenticated showfeature page add user buttons. So it causes the unauthenticated click button async function to timeout because that button is not visible (because only the authenticated page is visible). I fixed it by adding a check in the clickElement method and now it is not blocking it.
